client/react/frontpage/components/SocialPanel.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import './SocialPanel.scss';
+
+// update social media links here
+// img is the base name of the Font Awesome icon
+const socialMediaLinks = [
+  {
+    img: 'facebook',
+    link: 'https://www.facebook.com/uclaradio',
+  },
+  {
+    img: 'twitter',
+    link: 'https://twitter.com/UCLAradio',
+  },
+  {
+    img: 'instagram',
+    link: 'https://www.instagram.com/uclaradio/',
+  },
+  {
+    img: 'snapchat-ghost',
+    link: 'https://www.snapchat.com/add/uclaradio',
+  },
+];
+
+const SocialPanel = React.createClass({
```

Comment:
Let's convert this component into a [stateless functional component](https://medium.com/@joshblack/stateless-components-in-react-0-14-f9798f8b992d). I linked to a medium post that describes what it is and an example of how to do it. Essentially, our component is not holding any state and simply needs to return markup. 

If you end up following the guide I linked, don't worry about the part where he mentions `propTypes`. We don't have that yet. Let me know if you have any questions (especially with all the ES6 arrow function stuff).

Reply:
Refactored!